zim/gui/pageview/textbuffer.py

```diff
@@ -796,8 +797,12 @@ def _create_link_tag(self, text, href, **attrib):
 		if hasattr(href, 'uri'):
 			href = href.uri
 		assert isinstance(href, str) or href is None
-
-		tag = self.create_tag(None, **self.tag_styles['link'])
+		url: str = href or ''
+		is_external_link : bool = url.startswith('http://') or url.startswith('https://')
```

Comment:
This doesn't detect all external links, eg. `mailto:` links (see also: #513). Link parsing can get pretty complicated quite quickly so it'd be best to use our existing facilities: the `link_type` in `zim.parsing`. You can see how it is used here to display a different tool tip based on whether the link is internal or external:

https://github.com/zim-desktop-wiki/zim-desktop-wiki/blob/70355fa66c9b6abfbb15aec2675cbae55a6153fe/zim/gui/pageview/__init__.py#L879-L888

(You can also see another instance of anchors being anchors (see also: #1942))

Reply:
Thanks!
I've updated the PR, PTAL.